core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/BoxDerivedInfo.java

```diff
@@ -26,11 +31,28 @@
   @Temporal(TemporalType.TIMESTAMP)
   private Date lastModified;
 
+  @Temporal(TemporalType.TIMESTAMP)
+  private Date created;
+
+  @OneToOne(targetEntity = UserImpl.class)
+  @JoinColumn(name = "creator", nullable = false)
+  @JsonBackReference
+  private User creator;
```

Comment:
The BackReference here only makes sense if `User` has a "boxDerivedInfos" collection